flink-python/pom.xml

```diff
@@ -314,7 +314,7 @@ under the License.
 			<!-- Indirectly accessed in pyflink_gateway_server -->
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-sql-connector-aws-kinesis-firehose</artifactId>
-			<version>${project.version}</version>
+			<version>4.0.0-1.16</version>
```

Comment:
Why do we set the version explicitly here? Should it be:

`4.0.0-${project.version}`

Reply:
`${project.version}` includes the patch value `z` (`x.y.z`) and would be wrong (1.17 instead of 1.16).